packages/cf-component-card/src/CardBlock.js

```diff
@@ -1,13 +1,20 @@
 import React from 'react';
-
 import PropTypes from 'prop-types';
+import { createComponent } from 'cf-style-container';
+
+const styles = ({ theme }) => ({
+  padding: '1rem 1.4rem 1.26667rem',
+  borderTop: `1px solid ${theme.colorGrayLight}`,
```

Comment:
I can't find this in the existing CSS, only `.cf-card__block + .cf-card__block`.